app/controllers/users.js

```diff
@@ -269,3 +271,68 @@ exports.user = (req, res, next, id) => {
       next();
     });
 };
+
+/**
+ * Invite user to play game.
+ */
+const sgMail = require('@sendgrid/mail');
+
+sgMail.setApiKey('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');
```

Comment:
Please let's move this API key to `process.env.SENDGRID_API_KEY` @OlatunjiYso @Daramola98